New setDebugUtilsObjectNameEXT(handle, name) doesnt work with VULKAN_HPP_NO_STRUCT_CONSTRUCTORS defined

Hello, newly added `setDebugUtilsObjectNameEXT` (#2329 ) doesn't work when `VULKAN_HPP_NO_STRUCT_CONSTRUCTORS` is defined.
Compiler complaint:
```
In file included from /home/salio/CLionProjects/VoidScape/CoriEngine/Engine/thirdparty/Vulkan-Headers/include/vulkan/vulkan.hpp:9550,
                 from /home/salio/CLionProjects/VoidScape/CoriEngine/Engine/thirdparty/Vulkan-Headers/include/vulkan/vulkan_raii.hpp:11,
                 from /home/salio/CLionProjects/VoidScape/CoriEngine/Engine/src/Graphics/Vulkan/VulkanEngine.hpp:5:
/home/salio/CLionProjects/VoidScape/CoriEngine/Engine/thirdparty/Vulkan-Headers/include/vulkan/vulkan_funcs.hpp:15995:43: note: declared here
15995 |                                           Device::setDebugUtilsObjectNameEXT( HandleType const & handle, std::string const & name, Dispatch const & d ) const
      |                                           ^~~~~~
/home/salio/CLionProjects/VoidScape/CoriEngine/Engine/thirdparty/Vulkan-Headers/include/vulkan/vulkan_funcs.hpp: In instantiation of ‘vk::ResultValueType<void>::type vk::Device::setDebugUtilsObjectNameEXT(const HandleType&, const std::string&, const Dispatch&) const [with HandleType = vk::ImageView; Dispatch = vk::detail::DispatchLoaderDynamic; vk::ResultValueType<void>::type = vk::Result; std::string = std::__cxx11::basic_string<char>]’:
/home/salio/CLionProjects/VoidScape/CoriEngine/Engine/src/Graphics/Vulkan/VulkanEngine.cpp:470:40:   required from here
  470 |                                 m_Device.setDebugUtilsObjectNameEXT(view, name);
      |                                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~
/home/salio/CLionProjects/VoidScape/CoriEngine/Engine/thirdparty/Vulkan-Headers/include/vulkan/vulkan_funcs.hpp:15998:55: error: cannot convert ‘const vk::ObjectType’ to ‘vk::StructureType’ in initialization
15998 |     VULKAN_HPP_NAMESPACE::DebugUtilsObjectNameInfoEXT nameInfo(
      |                                                       ^~~~~~~~

```

Because this:
`vk::DebugUtilsObjectNameInfoEXT nameInfo(vk::ObjectType::eImageView, reinterpret_cast<uint64_t>( static_cast<VkImageView>( view ) ), name.c_str() );`
is not valid when using `VULKAN_HPP_NO_STRUCT_CONSTRUCTORS`, 
either this:
`vk::DebugUtilsObjectNameInfoEXT nameInfo(vk::StructureType::eDebugUtilsObjectNameInfoEXT, nullptr, vk::ObjectType::eImageView, reinterpret_cast<uint64_t>( static_cast<VkImageView>( view ) ), name.c_str() );`

or this:
```
vk::DebugUtilsObjectNameInfoEXT nameInfo{
	.objectType = vk::ObjectType::eImageView,
	.objectHandle = reinterpret_cast<uint64_t>( static_cast<VkImageView>( view ) ),
	.pObjectName = name.c_str()
};
```
is needed.

--- a/VulkanHppGenerator.cpp
+++ b/VulkanHppGenerator.cpp
@@ -1681,8 +1681,11 @@ void VulkanHppGenerator::extendSpecialCommands( std::string const & name, bool d
     VULKAN_HPP_INLINE typename ResultValueType<void>::type Device::setDebugUtilsObjectNameEXT( HandleType const & handle, std::string const & name ) const
     {
       static_assert( VULKAN_HPP_NAMESPACE::isVulkanHandleType<HandleType>::value, "HandleType must be a Vulkan handle type" );
-      VULKAN_HPP_NAMESPACE::DebugUtilsObjectNameInfoEXT nameInfo(
-        handle.objectType, reinterpret_cast<uint64_t>( static_cast<typename HandleType::CType>( handle ) ), name.c_str() );
+      // It might be, that neither constructors, nor setters, nor designated initializers are available... need to explicitly set member by member
+      VULKAN_HPP_NAMESPACE::DebugUtilsObjectNameInfoEXT nameInfo;
+      nameInfo.objectType   = handle.objectType;
+      nameInfo.objectHandle = reinterpret_cast<uint64_t>( static_cast<typename HandleType::CType>( handle ) );
+      nameInfo.pObjectName  = name.c_str();
       return setDebugUtilsObjectNameEXT( nameInfo );
     }
 )" );
@@ -1712,8 +1715,11 @@ void VulkanHppGenerator::extendSpecialCommands( std::string const & name, bool d
                                           Device::setDebugUtilsObjectNameEXT( HandleType const & handle, std::string const & name, Dispatch const & d ) const
   {
     static_assert( VULKAN_HPP_NAMESPACE::isVulkanHandleType<HandleType>::value, "HandleType must be a Vulkan handle type" );
-    VULKAN_HPP_NAMESPACE::DebugUtilsObjectNameInfoEXT nameInfo(
-      handle.objectType, reinterpret_cast<uint64_t>( static_cast<typename HandleType::CType>( handle ) ), name.c_str() );
+    // It might be, that neither constructors, nor setters, nor designated initializers are available... need to explicitly set member by member
+    VULKAN_HPP_NAMESPACE::DebugUtilsObjectNameInfoEXT nameInfo;
+    nameInfo.objectType   = handle.objectType;
+    nameInfo.objectHandle = reinterpret_cast<uint64_t>( static_cast<typename HandleType::CType>( handle ) );
+    nameInfo.pObjectName  = name.c_str();
     return setDebugUtilsObjectNameEXT( nameInfo, d );
   }
 )" );
@@ -1744,8 +1750,13 @@ void VulkanHppGenerator::extendSpecialCommands( std::string const & name, bool d
       Device::setDebugUtilsObjectTagEXT( HandleType const & handle, uint64_t name, TagType const & tag ) const
     {
       static_assert( VULKAN_HPP_NAMESPACE::isVulkanHandleType<HandleType>::value, "HandleType must be a Vulkan handle type" );
-      VULKAN_HPP_NAMESPACE::DebugUtilsObjectTagInfoEXT tagInfo(
-        handle.objectType, reinterpret_cast<uint64_t>( static_cast<typename HandleType::CType>( handle ) ), name, sizeof( TagType ), &tag );
+      // It might be, that neither constructors, nor setters, nor designated initializers are available... need to explicitly set member by member
+      VULKAN_HPP_NAMESPACE::DebugUtilsObjectTagInfoEXT tagInfo;
+      tagInfo.objectType   = handle.objectType;
+      tagInfo.objectHandle = reinterpret_cast<uint64_t>( static_cast<typename HandleType::CType>( handle ) );
+      tagInfo.tagName      = name;
+      tagInfo.tagSize      = sizeof( TagType );
+      tagInfo.pTag         = &tag;
       return setDebugUtilsObjectTagEXT( tagInfo );
     }
 )" );
@@ -1775,8 +1786,13 @@ void VulkanHppGenerator::extendSpecialCommands( std::string const & name, bool d
     Device::setDebugUtilsObjectTagEXT( HandleType const & handle, uint64_t name, TagType const & tag, Dispatch const & d ) const
   {
     static_assert( VULKAN_HPP_NAMESPACE::isVulkanHandleType<HandleType>::value, "HandleType must be a Vulkan handle type" );
-    VULKAN_HPP_NAMESPACE::DebugUtilsObjectTagInfoEXT tagInfo(
-      handle.objectType, reinterpret_cast<uint64_t>( static_cast<typename HandleType::CType>( handle ) ), name, sizeof( TagType ), &tag );
+    // It might be, that neither constructors, nor setters, nor designated initializers are available... need to explicitly set member by member
+    VULKAN_HPP_NAMESPACE::DebugUtilsObjectTagInfoEXT tagInfo;
+    tagInfo.objectType   = handle.objectType;
+    tagInfo.objectHandle = reinterpret_cast<uint64_t>( static_cast<typename HandleType::CType>( handle ) );
+    tagInfo.tagName      = name;
+    tagInfo.tagSize      = sizeof( TagType );
+    tagInfo.pTag         = &tag;
     return setDebugUtilsObjectTagEXT( tagInfo, d );
   }
 )" );

--- a/vulkan/vulkan.cppm
+++ b/vulkan/vulkan.cppm
@@ -9365,7 +9365,8 @@ export namespace std
 #endif
 }  // namespace std
 
-export {
+export
+{
   // This VkFlags type is used as part of a bitfield in some structures.
   // As it can't be mimicked by vk-data types, we need to export just that.
   using ::VkGeometryInstanceFlagsKHR;

--- a/vulkan/vulkan_funcs.hpp
+++ b/vulkan/vulkan_funcs.hpp
@@ -15995,8 +15995,11 @@ namespace VULKAN_HPP_NAMESPACE
                                           Device::setDebugUtilsObjectNameEXT( HandleType const & handle, std::string const & name, Dispatch const & d ) const
   {
     static_assert( VULKAN_HPP_NAMESPACE::isVulkanHandleType<HandleType>::value, "HandleType must be a Vulkan handle type" );
-    VULKAN_HPP_NAMESPACE::DebugUtilsObjectNameInfoEXT nameInfo(
-      handle.objectType, reinterpret_cast<uint64_t>( static_cast<typename HandleType::CType>( handle ) ), name.c_str() );
+    // It might be, that neither constructors, nor setters, nor designated initializers are available... need to explicitly set member by member
+    VULKAN_HPP_NAMESPACE::DebugUtilsObjectNameInfoEXT nameInfo;
+    nameInfo.objectType   = handle.objectType;
+    nameInfo.objectHandle = reinterpret_cast<uint64_t>( static_cast<typename HandleType::CType>( handle ) );
+    nameInfo.pObjectName  = name.c_str();
     return setDebugUtilsObjectNameEXT( nameInfo, d );
   }
 #endif /* VULKAN_HPP_DISABLE_ENHANCED_MODE */
@@ -16034,8 +16037,13 @@ namespace VULKAN_HPP_NAMESPACE
     Device::setDebugUtilsObjectTagEXT( HandleType const & handle, uint64_t name, TagType const & tag, Dispatch const & d ) const
   {
     static_assert( VULKAN_HPP_NAMESPACE::isVulkanHandleType<HandleType>::value, "HandleType must be a Vulkan handle type" );
-    VULKAN_HPP_NAMESPACE::DebugUtilsObjectTagInfoEXT tagInfo(
-      handle.objectType, reinterpret_cast<uint64_t>( static_cast<typename HandleType::CType>( handle ) ), name, sizeof( TagType ), &tag );
+    // It might be, that neither constructors, nor setters, nor designated initializers are available... need to explicitly set member by member
+    VULKAN_HPP_NAMESPACE::DebugUtilsObjectTagInfoEXT tagInfo;
+    tagInfo.objectType   = handle.objectType;
+    tagInfo.objectHandle = reinterpret_cast<uint64_t>( static_cast<typename HandleType::CType>( handle ) );
+    tagInfo.tagName      = name;
+    tagInfo.tagSize      = sizeof( TagType );
+    tagInfo.pTag         = &tag;
     return setDebugUtilsObjectTagEXT( tagInfo, d );
   }
 #endif /* VULKAN_HPP_DISABLE_ENHANCED_MODE */

--- a/vulkan/vulkan_raii.hpp
+++ b/vulkan/vulkan_raii.hpp
@@ -21153,8 +21153,11 @@ namespace VULKAN_HPP_NAMESPACE
     VULKAN_HPP_INLINE typename ResultValueType<void>::type Device::setDebugUtilsObjectNameEXT( HandleType const & handle, std::string const & name ) const
     {
       static_assert( VULKAN_HPP_NAMESPACE::isVulkanHandleType<HandleType>::value, "HandleType must be a Vulkan handle type" );
-      VULKAN_HPP_NAMESPACE::DebugUtilsObjectNameInfoEXT nameInfo(
-        handle.objectType, reinterpret_cast<uint64_t>( static_cast<typename HandleType::CType>( handle ) ), name.c_str() );
+      // It might be, that neither constructors, nor setters, nor designated initializers are available... need to explicitly set member by member
+      VULKAN_HPP_NAMESPACE::DebugUtilsObjectNameInfoEXT nameInfo;
+      nameInfo.objectType   = handle.objectType;
+      nameInfo.objectHandle = reinterpret_cast<uint64_t>( static_cast<typename HandleType::CType>( handle ) );
+      nameInfo.pObjectName  = name.c_str();
       return setDebugUtilsObjectNameEXT( nameInfo );
     }
 
@@ -21177,8 +21180,13 @@ namespace VULKAN_HPP_NAMESPACE
       Device::setDebugUtilsObjectTagEXT( HandleType const & handle, uint64_t name, TagType const & tag ) const
     {
       static_assert( VULKAN_HPP_NAMESPACE::isVulkanHandleType<HandleType>::value, "HandleType must be a Vulkan handle type" );
-      VULKAN_HPP_NAMESPACE::DebugUtilsObjectTagInfoEXT tagInfo(
-        handle.objectType, reinterpret_cast<uint64_t>( static_cast<typename HandleType::CType>( handle ) ), name, sizeof( TagType ), &tag );
+      // It might be, that neither constructors, nor setters, nor designated initializers are available... need to explicitly set member by member
+      VULKAN_HPP_NAMESPACE::DebugUtilsObjectTagInfoEXT tagInfo;
+      tagInfo.objectType   = handle.objectType;
+      tagInfo.objectHandle = reinterpret_cast<uint64_t>( static_cast<typename HandleType::CType>( handle ) );
+      tagInfo.tagName      = name;
+      tagInfo.tagSize      = sizeof( TagType );
+      tagInfo.pTag         = &tag;
       return setDebugUtilsObjectTagEXT( tagInfo );
     }
 

